version.txt

```diff
@@ -0,0 +1 @@
+2.1
```

Comment:
i think this file needs to be removed. we should write to version.txt only when we release. there might be a bug in the script. i'll take a look.

Reply:
ah got it, this file should not be there, ok. I'll remove it.